.github/styles/config/vocabularies/Base/accept.txt

```diff
@@ -95,6 +95,7 @@ Mockbin
 Moesif
 MongoDB
 Multus
+mTLS
```

Comment:
Argh this is in multiple PRs sorry. We need one entry I think

```
m?(TLS|tls)
```

Though I guess remove lowercase if you want

Reply:
I'd remove lowercase because we only allow TLS. Coherency is just easier :)

Reply:
yeah i meant more important is the `m?TLS` part otherwise I think it will mismatch mTLS as TLS depending on which one comes first. At least it seemed like that in https://github.com/kumahq/kuma-website/pull/1967 but we can also add it in a separate PR